gio/src/auto/unix_mount_entry.rs

```diff
@@ -3,17 +3,227 @@
+        unsafe {
+            from_glib_full(ffi::g_unix_mount_entry_guess_name(
+                self.to_glib_none_mut().0,
+            ))
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_guess_should_display")]
+    pub fn guess_should_display(&mut self) -> bool {
+        unsafe {
+            from_glib(ffi::g_unix_mount_entry_guess_should_display(
+                self.to_glib_none_mut().0,
+            ))
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_guess_symbolic_icon")]
+    pub fn guess_symbolic_icon(&mut self) -> Icon {
+        unsafe {
+            from_glib_full(ffi::g_unix_mount_entry_guess_symbolic_icon(
+                self.to_glib_none_mut().0,
+            ))
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_is_readonly")]
+    pub fn is_readonly(&mut self) -> bool {
+        unsafe {
+            from_glib(ffi::g_unix_mount_entry_is_readonly(
+                self.to_glib_none_mut().0,
+            ))
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_is_system_internal")]
+    pub fn is_system_internal(&mut self) -> bool {
+        unsafe {
+            from_glib(ffi::g_unix_mount_entry_is_system_internal(
+                self.to_glib_none_mut().0,
+            ))
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_at")]
+    pub fn at(mount_path: impl AsRef<std::path::Path>) -> (Option<UnixMountEntry>, u64) {
+        unsafe {
+            let mut time_read = std::mem::MaybeUninit::uninit();
+            let ret = from_glib_full(ffi::g_unix_mount_entry_at(
+                mount_path.as_ref().to_glib_none().0,
+                time_read.as_mut_ptr(),
+            ));
+            (ret, time_read.assume_init())
+        }
+    }
+
+    #[cfg(feature = "v2_84")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+    #[doc(alias = "g_unix_mount_entry_for")]
+    #[doc(alias = "for")]
+    pub fn for_(file_path: impl AsRef<std::path::Path>) -> (Option<UnixMountEntry>, u64) {
+        unsafe {
+            let mut time_read = std::mem::MaybeUninit::uninit();
+            let ret = from_glib_full(ffi::g_unix_mount_entry_for(
+                file_path.as_ref().to_glib_none().0,
+                time_read.as_mut_ptr(),
+            ));
+            (ret, time_read.assume_init())
+        }
+    }
+}
+
+#[cfg(feature = "v2_84")]
+#[cfg_attr(docsrs, doc(cfg(feature = "v2_84")))]
+impl PartialEq for UnixMountEntry {
+    #[inline]
+    fn eq(&self, other: &Self) -> bool {
+        self.compare(other) == 0
+    }
+}
+
+impl Eq for UnixMountEntry {}
```

Comment:
This will fail as PartialEq is behind a guard, maybe we should fix gir here:)

Reply:
Yes, but we can have `PartialEq` without the guard too :)